libgeopmd/src/NVMLIOGroup.cpp

```diff
@@ -124,6 +133,7 @@ namespace geopm
                                   "GPU PCIE receive throughput in bytes per second over a 20 millisecond period",
                                   {},
                                   GEOPM_DOMAIN_GPU,
+                                  M_UNITS_HERTZ,
```

Comment:
Hertz isn't really throughput... do we want a new unit for throughput? M_UNITS_BPS ?

Reply:
We're going to table this until we can query it programmatically.  Then we'll revisit what the proper units are here.